### PR TITLE
Multiples JmapReadOnlyScenario fixes

### PR DESCRIPTION
**Pull request by blackheaven**

### CODE REVIEW

File: src/test/scala-2.12/org/apache/james/gatling/simulation/jmap/JmapReadOnlySimulation.scala [hunk 1]
```diff
@@ -15,17 +15,16 @@ class JmapReadOnlySimulation extends Simulation {
...
         username = Username(recordValueToString(record("username"))),
         jwtAccessToken = JwtAccessToken(recordValueToString(record("jwtAccessToken")))))
-    .toIterator
 
-  val feeder: AuthenticatedUserFeederBuilder = AuthenticatedUserFeeder.toFeeder(authenticatedUsers)
+  val feeder: AuthenticatedUserFeederBuilder = AuthenticatedUserFeeder.toFeeder(authenticatedUsers.toIterator)
 
   setUp(new JmapReadOnlyScenario().generate(feeder, Configuration.ScenarioDuration)
-    .inject(UsersTotal(Configuration.UserCount).injectDuring(Configuration.InjectionDuration))
+    .inject(UsersTotal(authenticatedUsers.length).injectDuring(Configuration.InjectionDuration))
```
Comment (rouazana) [inline src/test/scala-2.12/org/apache/james/gatling/simulation/jmap/JmapReadOnlySimulation.scala:27]: does it still work with a circular feeder? (but maybe it's not needed for this particular simulation)

**Review by Arsnael — APPROVED** [review]

File: src/main/scala-2.12/org/apache/james/gatling/jmap/scenari/JmapReadOnlyScenario.scala [hunk 2]
```diff
@@ -68,7 +72,8 @@ class JmapReadOnlyScenario {
       .feed(userFeeder)
       .during(duration) {
         group(InboxHomeLoading.name)(
-          doIf(shouldPoll) {
+          pause(1 second)
```
Comment (chibenwa) [inline src/main/scala-2.12/org/apache/james/gatling/jmap/scenari/JmapReadOnlyScenario.scala:75]: We should have a 30s pause to match inbox behaviour

File: src/main/scala-2.12/org/apache/james/gatling/jmap/scenari/JmapReadOnlyScenario.scala [hunk 3]
```diff
@@ -68,7 +72,8 @@ class JmapReadOnlyScenario {
...
+          .doIf(shouldPoll) {
             exec(poll)
           }
             .exec(Queries.getMailboxes)
```
Comment (chibenwa) [inline src/main/scala-2.12/org/apache/james/gatling/jmap/scenari/JmapReadOnlyScenario.scala:79]: We should not excute these queries if we had been polling. Is it the case?

File: src/test/scala-2.12/org/apache/james/gatling/simulation/jmap/JmapReadOnlySimulation.scala (hunk 1 above)
Comment (chibenwa) [reply src/test/scala-2.12/org/apache/james/gatling/simulation/jmap/JmapReadOnlySimulation.scala:27]: Yes circular feeder don't makes sense for this one.

**Review by chibenwa — COMMENTED** [review]
:+1: 

(thanks)

**Review by chibenwa — APPROVED** [review]

**Review by rouazana — APPROVED** [review]

**Review by Arsnael — APPROVED** [review]

**Pull request merged**